modules/atomic-widgets/assets/js/editor/module.js

```diff
@@ -1,8 +1,19 @@
 import Component from './component';
+import { AtomicWidgetType } from './atomic-widget-type';
 
 class Module extends elementorModules.editor.utils.Module {
 	onInit() {
 		$e.components.register( new Component() );
+
+		this.registerAtomicWidgetTypes();
+	}
+
+	registerAtomicWidgetTypes() {
+		Object.entries( elementor.widgetsCache ?? {} )
+			.filter( ( [ , widget ] ) => !! widget.atomic )
+			.forEach(
+				( [ type ] ) => elementor.elementsManager.registerElementType( new AtomicWidgetType( type ) ),
```

Comment:
@Nevoss Does every single widget have to register it's own "element type"? 🤔

Reply:
We can do it, but we have to change the `core` functionality of the `elementsManager` and I did not want to change much in the `core`.

Reply:
Yeah, did not find another way.

Reply:
Yes it should works, but I am not sure it is safe to treat the `atomic-widget` as a new element type. but maybe it is fine.

Reply:
@Nevoss Can you please try it for now and see if it works fine? I feel better about one new element type than multiple.